src/assets/stylesheets/extensions/_nav-multistep.scss

```diff
@@ -0,0 +1,154 @@
+@import "../config";
+
+.nav-multistep {
+  margin-top: 2rem;
+
+  ol {
+    display: flex;
+    margin-left: 0;
+    margin-bottom: 1rem;
+
+    li {
+      position:relative;
+      margin: 0;
+      list-style-type: none;
+      background: $codacy-neutral-800;
+      line-height: 1.25;
+
+      a,
+      p {
+        display: flex;
+        align-items: center;
+        width: 100%;
+        padding: 0 1rem;
+        color: #FFFFFF;
+      }
+
+      &.nav-multistep__step--active {
+        background: $codacy-neutral-300;
+
+        p {
+          font-weight: 700;
+          color: $codacy-neutral-800;
+        }
+      }
+
+      &:not(.nav-multistep__step--active):hover {
+        background: $codacy-neutral-700;
+      }
+    }
+  }
+}
+
+@include break-to-device(mobile landscape) {
+  .nav-multistep {
+    ol {
+      flex-direction: column;
+      li {
+        a,
+        p {
+          padding: 0.5rem 1rem;
+          margin: 0;
+        }
+
+        &:first-child,
+        &:nth-child(2) {
+          margin-bottom: 0.125rem;
+
+          &::after {
+            border-top: 0.375rem solid #FFFFFF;
+            border-right: 0.375rem solid transparent;
+            border-left: 0.375rem solid transparent;
+            position: absolute;
+            display: block;
+            content: "";
+            bottom: -0.5rem;
+            left: 1.5rem;
+            z-index: 1;
+          }
+        }
+      }
+    }
+  }
+}
+
+@include break-from-device(tablet portrait) {
+  .nav-multistep {
+    ol {
+      li {
+        display: flex;
+        flex: 0 0 33%;
+        height: 3rem;
+
+        &.nav-multistep__step--active {
+          &:first-child,
+          &:nth-child(2) {
+            &::after {
+              border-left-color: $codacy-neutral-300 !important;
+            }
+          }
+
+          &:nth-child(2),
+          &:nth-child(3) {
```

Comment:
you are assuming you will always have only 3 steps right? This might not be the case if you decide to use this somewhere else 🤔 
I would suggest looking into the css suffix like :odd or :even

Or even use the logic "the default design is this one for the 2 and 3, I just need to change the first-child, so in the first child I can remove this borders"

Reply:
Thanks @inesgomas! Good point 👍 

I did assume to have only 3 steps for now, but this would have become tech debt in the future. I'll apply your suggestion.